packages/template-retail-react-app/config/default.js

```diff
@@ -22,7 +22,8 @@ module.exports = {
             },
             social: {
                 enabled: false,
-                idps: ['google', 'apple']
+                idps: ['google', 'apple'],
+                redirectURI: '/social-callback'
```

Comment:
I have added the `redirectURI` here so that users can configure the redirectURI they want to use specifically for social login. This may differ from the redirectURI they have configured in `AppConfig` [here](https://github.com/SalesforceCommerceCloud/pwa-kit/blob/develop/packages/template-retail-react-app/app/components/_app-config/index.jsx#L64)